src/redirects.js

```diff
@@ -0,0 +1,18 @@
+import fetch from 'node-fetch';
+
+export async function checkRedirect (url) {
+  const options = {
+    method: 'GET',
+    redirect: 'follow'
+  };
+  const response = await fetch(url, options);
+  const path = {
+    isRedirected: response.redirected,
```

Comment:
Right now, since we use DOI links, there will always be at least one redirect experienced by the user. At the same time, I understand why DOI links are preferred as they will always link back to the paper we need, while URLs might change over time. Do we want to keep using DOIs here even though they cause redirects?

Reply:
We should store both the original DOI and the 'final' link.  View the final link as a cached value, with the DOI as the original source.

Reply:
I see, what's the difference between the cached value and original source when displayed on the dashboard? How do they interact with one another? Would the original source be the link on the dashboard, which leads immediately to the cached value?